pkg/telemetryservice/sql/tdengine/tdengine.go

```diff
@@ -67,3 +118,40 @@ func (db *DBHelper) insertDataToDB(ctx context.Context, rawData []byte) error {
 func constructTDengineUri(sqlcs *v1alpha1.SQLConnectionSetting) string {
 	return fmt.Sprintf("%s:%s@http(%s)/%s", *sqlcs.UserName, *sqlcs.Secret, *sqlcs.ServerAddress, *sqlcs.DBName)
 }
+
+func (db *DBHelper) query(querySql string) ([]*StructureData, error) {
+	StructureDatas := make([]*StructureData, 0)
+	rows, err := db.DB.Query(querySql)
+	if err != nil {
+		logger.Errorf("Error to query data from tdengine, sql code: %s", querySql)
```

Comment:
logger.Errorf("Error to query data from tdengine, sql: %s, error: %s", querySql, err.Error())